pyspark-notebook/test/test_spark.py

```diff
@@ -9,22 +9,18 @@
 def test_spark_shell(container):
     """Checking if Spark (spark-shell) is running properly"""
     c = container.run(
-        tty=True,
-        command=['start.sh', 'bash', '-c', 'spark-shell <<< "1+1"']
+        tty=True, command=["start.sh", "bash", "-c", 'spark-shell <<< "1+1"']
```

Comment:
You think? Hmmm, I think it was worse for the same reason you thought https://github.com/jupyter/docker-stacks/pull/1387/files#r659231456 was worse.

Anyhow, in this situation, one can add a trailing comma after the `command` argument and it would go back to how it was before if that is desired.

Reply:
Let's add comma and have every argument on a separate line :)

Reply:
Done!